docs/config/tech-docs.yml

```diff
@@ -0,0 +1,38 @@
+# Host to use for canonical URL generation (without trailing slash)
+host: api2.publish-teacher-training-courses.service.gov.uk
+
+# Header-related options
+show_govuk_logo: true
+service_name: Teacher Training Courses
+service_link: https://www.find-postgraduate-teacher-training.service.gov.uk/
+phase: Beta
+
+# Links to show on right-hand-side of header
+header_links:
+  About: /
+  Support: /support.html
+
+# Enables search functionality. This indexes pages only and is not recommended for single-page sites.
+enable_search: true
+
+# Tracking ID from Google Analytics (e.g. UA-XXXX-Y)
+ga_tracking_id:
+
+# Enable multipage navigation in the sidebar
+multipage_nav: true
+
+# Enable collapsible navigation in the sidebar
+collapsible_nav: true
+
+# Table of contents depth – how many levels to include in the table of contents.
+# If your ToC is too long, reduce this number and we'll only show higher-level
+# headings.
+max_toc_heading_level: 6
+
+# Prevent robots from indexing (e.g. whilst in development)
+prevent_indexing: false
+
+show_contribution_banner: true
+github_repo: DFE-Digital/teacher-training-api
+
+api_path: https://api2.publish-teacher-training-courses.service.gov.uk/api-docs/v1/swagger.json
```

Comment:
This is grabbing the API spec from the running server, we should probably use `../swagger/v1/swagger.json`.

Reply:
Incidentally, we should make more effort to make it clear that this isn't the existing `v1`, and I'm not sure that the filename `swagger` is really descriptive. How about we use the directory `swagger/public-v1/api_spec.json` or something like that?

Reply:
ok, i've made the suggested changes